rules/method-2-using-file-reference-assemblies-within-vsnet/rule.md

```diff
@@ -0,0 +1,32 @@
+---
+type: rule
+title: "Method #2 - Using File Reference Assemblies within VS.NET"
+uri: method-2-using-file-reference-assemblies-within-vsnet
```

Comment:
@bradystroud @fenix2222 @tiagov8 

Can't these "Method 1,2,3,4" just be merged into a single rule? 

But also these are surely archived rules right?

Reply:
Merged in here https://github.com/SSWConsulting/SSW.Rules.Content/pull/8361